AddonExample/AddonExample.csproj

```diff
@@ -106,7 +106,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json">
-      <Version>12.0.3</Version>
+      <Version>13.0.1</Version>
```

Comment:
This change can be moved to another pull request if it makes sense. This is to solve the following error:

```
"C:\OverlayPlugin\OverlayPlugin.sln" (default target) (1) ->
"C:\OverlayPlugin\AddonExample\AddonExample.csproj" (default target) (6) ->
(CoreCompile target) ->
  CSC : error CS1705: Assembly 'OverlayPlugin.Common' with identity 'OverlayPlugin.Common, Version=0.18.2.0, Culture=neutral, PublicKeyToken=null' uses 'Newtonsoft.Json, Version=13.0.0.0, Culture=neutra
l, PublicKeyToken=30ad4fe6b2a6aeed' which has a higher version than referenced assembly 'Newtonsoft.Json' with identity 'Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6ae
ed' [C:\OverlayPlugin\AddonExample\AddonExample.csproj]
  CSC : error CS1705: Assembly 'OverlayPlugin.Core' with identity 'OverlayPlugin.Core, Version=0.18.2.0, Culture=neutral, PublicKeyToken=null' uses 'Newtonsoft.Json, Version=13.0.0.0, Culture=neutral, P
ublicKeyToken=30ad4fe6b2a6aeed' which has a higher version than referenced assembly 'Newtonsoft.Json' with identity 'Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed'
[C:\OverlayPlugin\AddonExample\AddonExample.csproj]
```